src/casp/services/api/services/cellarium_general_service.py

```diff
@@ -23,6 +26,18 @@ def get_application_info(self) -> schemas.ApplicationInfo:
         """
         return self.cellarium_general_dm.get_application_info()
 
+    def validate_client_version(self, client_version: str):
```

Comment:
I think we want a positive and negative unit test for this method at least

Reply:
Ooo good idea